modules/openapi-generator/src/main/resources/JavaSpring/converter.mustache

```diff
@@ -0,0 +1,34 @@
+package org.openapitools.configuration;
+
+{{#models}}
+    {{#model}}
+        {{#isEnum}}
+import {{modelPackage}}.{{name}};
+        {{/isEnum}}
+    {{/model}}
+{{/models}}
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.convert.converter.Converter;
+
+@Configuration
+public class EnumConverters {
+
+{{#models}}
+{{#model}}
+{{#isEnum}}
+    @Bean
+    Converter<String, {{name}}> {{name}}Converter() {
+        return new Converter<String, {{name}}>() {
+            @Override
+            public {{name}} convert(String source) {
+                return {{name}}.fromValue(source);
```

Comment:
enums can have other data type except String. Note `{{dataType}}` for `fromValu` param
https://github.com/OpenAPITools/openapi-generator/blob/f3fa6758dfbcf4e1ea38dce20389bdb262f699b9/modules/openapi-generator/src/main/resources/JavaSpring/enumClass.mustache#L42

Reply:
Ok, I've done the change.